<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>human_id</th>
      <th>wikidata_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>The top 100 papers</td>
      <td> <a href=./notes/Q28250597.md> Q28250597 </a></td>
    </tr>
    <tr>
      <th>1</th>
      <td>The origin and evolution of cell types</td>
      <td> <a href=./notes/Q38798014.md> Q38798014 </a></td>
    </tr>
    <tr>
      <th>2</th>
      <td>Make code accessible with these cloud services</td>
      <td> <a href=./notes/Q91130546.md> Q91130546 </a></td>
    </tr>
    <tr>
      <th>3</th>
      <td>Genetic address book for retinal cell types.</td>
      <td> <a href=./notes/Q43299861.md> Q43299861 </a></td>
    </tr>
    <tr>
      <th>4</th>
      <td>A community-based transcriptomics classification and nomenclature of neocortical cell types</td>
      <td> <a href=./notes/Q98665291.md> Q98665291 </a></td>
    </tr>
    <tr>
      <th>5</th>
      <td>The network effect: studying COVID-19 pathology with the Human Cell Atlas</td>
      <td> <a href=./notes/Q96817154.md> Q96817154 </a></td>
    </tr>
    <tr>
      <th>6</th>
      <td>Comprehensive Classification of Retinal Bipolar Neurons by Single-Cell Transcriptomics.</td>
      <td> <a href=./notes/Q40967362.md> Q40967362 </a></td>
    </tr>
    <tr>
      <th>7</th>
      <td>Identifiers for the 21st century: How to design, provision, and reuse persistent identifiers to maximize utility and impact of life science data</td>
      <td> <a href=./notes/Q33037209.md> Q33037209 </a></td>
    </tr>
    <tr>
      <th>8</th>
      <td>The human disease network</td>
      <td> <a href=./notes/Q24678240.md> Q24678240 </a></td>
    </tr>
    <tr>
      <th>9</th>
      <td>The evolving concept of cell identity in the single cell era</td>
      <td> <a href=./notes/Q93086971.md> Q93086971 </a></td>
    </tr>
    <tr>
      <th>10</th>
      <td>PRINTS and its automatic supplement, prePRINTS.</td>
      <td> <a href=./notes/Q34169535.md> Q34169535 </a></td>
    </tr>
    <tr>
      <th>11</th>
      <td>The naming of neurons: applications of taxonomic theory to the study of cellular populations.</td>
      <td> <a href=./notes/Q34065481.md> Q34065481 </a></td>
    </tr>
    <tr>
      <th>12</th>
      <td>Making your database available through Wikipedia: the pros and cons</td>
      <td> <a href=./notes/Q28254676.md> Q28254676 </a></td>
    </tr>
    <tr>
      <th>13</th>
      <td>ezTag: tagging biomedical concepts via interactive learning.</td>
      <td> <a href=./notes/Q55483834.md> Q55483834 </a></td>
    </tr>
    <tr>
      <th>14</th>
      <td>GNormPlus: An Integrative Approach for Tagging Genes, Gene Families, and Protein Domains</td>
      <td> <a href=./notes/Q28200510.md> Q28200510 </a></td>
    </tr>
    <tr>
      <th>15</th>
      <td>tmVar 2.0: integrating genomic variant information from literature with dbSNP and ClinVar for precision medicine.</td>
      <td> <a href=./notes/Q47317210.md> Q47317210 </a></td>
    </tr>
    <tr>
      <th>16</th>
      <td>Assisting manual literature curation for protein-protein interactions using BioQRator</td>
      <td> <a href=./notes/Q33928865.md> Q33928865 </a></td>
    </tr>
    <tr>
      <th>17</th>
      <td>Neuronal diversity and convergence in a visual system developmental atlas</td>
      <td> <a href=./notes/Q101226729.md> Q101226729 </a></td>
    </tr>
    <tr>
      <th>18</th>
      <td>Enabling Open Science: Wikidata for Research (Wiki4R)</td>
      <td> <a href=./notes/Q26707522.md> Q26707522 </a></td>
    </tr>
    <tr>
      <th>19</th>
      <td>What Is it Like to Be a Bat?</td>
      <td> <a href=./notes/Q2703462.md> Q2703462 </a></td>
    </tr>
    <tr>
      <th>20</th>
      <td>Ten tips for a text-mining-ready article: How to improve automated discoverability and interpretability</td>
      <td> <a href=./notes/Q96032351.md> Q96032351 </a></td>
    </tr>
    <tr>
      <th>21</th>
      <td>Jointly defining cell types from multiple single-cell datasets using LIGER</td>
      <td> <a href=./notes/Q100526347.md> Q100526347 </a></td>
    </tr>
    <tr>
      <th>22</th>
      <td>Dovetailing biology and chemistry: integrating the Gene Ontology with the ChEBI chemical ontology.</td>
      <td> <a href=./notes/Q34878068.md> Q34878068 </a></td>
    </tr>
    <tr>
      <th>23</th>
      <td>Numerical taxonomy.</td>
      <td> <a href=./notes/Q50735370.md> Q50735370 </a></td>
    </tr>
    <tr>
      <th>24</th>
      <td>A Taxonomic Problem</td>
      <td> <a href=./notes/Q60137015.md> Q60137015 </a></td>
    </tr>
    <tr>
      <th>25</th>
      <td>Pathway information extracted from 25 years of pathway figures</td>
      <td> <a href=./notes/Q101473819.md> Q101473819 </a></td>
    </tr>
    <tr>
      <th>26</th>
      <td>The Development of Taxonomic Theory Since 1851</td>
      <td> <a href=./notes/Q59058260.md> Q59058260 </a></td>
    </tr>
    <tr>
      <th>27</th>
      <td>Semantic integration of gene expression analysis tools and data sources using software connectors.</td>
      <td> <a href=./notes/Q30570244.md> Q30570244 </a></td>
    </tr>
    <tr>
      <th>28</th>
      <td>The principles of science : a treatise on logic and scientific method</td>
      <td> <a href=./notes/Q51427139.md> Q51427139 </a></td>
    </tr>
    <tr>
      <th>29</th>
      <td>Some thoughts on bacterial classification</td>
      <td> <a href=./notes/Q74657187.md> Q74657187 </a></td>
    </tr>
    <tr>
      <th>30</th>
      <td>Cross-linking BioThings APIs through JSON-LD to facilitate knowledge exploration.</td>
      <td> <a href=./notes/Q49483648.md> Q49483648 </a></td>
    </tr>
    <tr>
      <th>31</th>
      <td>Knowledge-based biomedical Data Science</td>
      <td> <a href=./notes/Q57001498.md> Q57001498 </a></td>
    </tr>
    <tr>
      <th>32</th>
      <td>The Species Concept Among Viruses</td>
      <td> <a href=./notes/Q59063943.md> Q59063943 </a></td>
    </tr>
    <tr>
      <th>33</th>
      <td>The application of computers to taxonomy</td>
      <td> <a href=./notes/Q28181838.md> Q28181838 </a></td>
    </tr>
    <tr>
      <th>34</th>
      <td>Publishing DisGeNET as nanopublications</td>
      <td> <a href=./notes/Q31194033.md> Q31194033 </a></td>
    </tr>
    <tr>
      <th>35</th>
      <td>The moral status of animals</td>
      <td> <a href=./notes/Q79763269.md> Q79763269 </a></td>
    </tr>
    <tr>
      <th>36</th>
      <td>The use of models in bacterial classification</td>
      <td> <a href=./notes/Q78658697.md> Q78658697 </a></td>
    </tr>
    <tr>
      <th>37</th>
      <td>The types of retinal ganglion cells: current status and implications for neuronal classification.</td>
      <td> <a href=./notes/Q38432885.md> Q38432885 </a></td>
    </tr>
    <tr>
      <th>38</th>
      <td>Genuine semantic publishing</td>
      <td> <a href=./notes/Q46412887.md> Q46412887 </a></td>
    </tr>
    <tr>
      <th>39</th>
      <td>The anatomy of a nanopublication</td>
      <td> <a href=./notes/Q57011346.md> Q57011346 </a></td>
    </tr>
    <tr>
      <th>40</th>
      <td>Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types.</td>
      <td> <a href=./notes/Q52720499.md> Q52720499 </a></td>
    </tr>
    <tr>
      <th>41</th>
      <td>A gene expression atlas of the central nervous system based on bacterial artificial chromosomes</td>
      <td> <a href=./notes/Q29617753.md> Q29617753 </a></td>
    </tr>
    <tr>
      <th>42</th>
      <td>SourceData: a semantic platform for curating and searching figures.</td>
      <td> <a href=./notes/Q48094767.md> Q48094767 </a></td>
    </tr>
    <tr>
      <th>43</th>
      <td>Biological knowledge management: the emerging role of the Semantic Web technologies.</td>
      <td> <a href=./notes/Q37490532.md> Q37490532 </a></td>
    </tr>
    <tr>
      <th>44</th>
      <td>Systematic integration of biomedical knowledge prioritizes drugs for repurposing.</td>
      <td> <a href=./notes/Q42255083.md> Q42255083 </a></td>
    </tr>
    <tr>
      <th>45</th>
      <td>Utopia documents: linking scholarly literature with research data</td>
      <td> <a href=./notes/Q28292614.md> Q28292614 </a></td>
    </tr>
    <tr>
      <th>46</th>
      <td>Astrocyte layers in the mammalian cerebral cortex revealed by a single-cell in situ transcriptomic map</td>
      <td> <a href=./notes/Q90591926.md> Q90591926 </a></td>
    </tr>
    <tr>
      <th>47</th>
      <td>Web scraping technologies in an API world</td>
      <td> <a href=./notes/Q86694255.md> Q86694255 </a></td>
    </tr>
    <tr>
      <th>48</th>
      <td>How we learnt to stop worrying and love web scraping</td>
      <td> <a href=./notes/Q99234140.md> Q99234140 </a></td>
    </tr>
    <tr>
      <th>49</th>
      <td>A translational profiling approach for the molecular characterization of CNS cell types</td>
      <td> <a href=./notes/Q29615252.md> Q29615252 </a></td>
    </tr>
    <tr>
      <th>50</th>
      <td>Publishing on the semantic web</td>
      <td> <a href=./notes/Q28185770.md> Q28185770 </a></td>
    </tr>
    <tr>
      <th>51</th>
      <td>A periodic table of cell types</td>
      <td> <a href=./notes/Q93086975.md> Q93086975 </a></td>
    </tr>
    <tr>
      <th>52</th>
      <td>Universal cell type identifier based on number theory.</td>
      <td> <a href=./notes/Q52374237.md> Q52374237 </a></td>
    </tr>
    <tr>
      <th>53</th>
      <td>Neuronal cell types.</td>
      <td> <a href=./notes/Q35830509.md> Q35830509 </a></td>
    </tr>
    <tr>
      <th>54</th>
      <td>Heterogeneity of neutrophils</td>
      <td> <a href=./notes/Q91997728.md> Q91997728 </a></td>
    </tr>
    <tr>
      <th>55</th>
      <td>Natural killer cell immunotypes related to COVID-19 disease severity</td>
      <td> <a href=./notes/Q98622035.md> Q98622035 </a></td>
    </tr>
    <tr>
      <th>56</th>
      <td>Adventures in semantic publishing: exemplar semantic enhancements of a research article</td>
      <td> <a href=./notes/Q21092566.md> Q21092566 </a></td>
    </tr>
    <tr>
      <th>57</th>
      <td>Reproducibility of Flow Cytometry Through Standardization: Opportunities and Challenges</td>
      <td> <a href=./notes/Q90588777.md> Q90588777 </a></td>
    </tr>
    <tr>
      <th>58</th>
      <td>Application of a translational profiling approach for the comparative analysis of CNS cell types</td>
      <td> <a href=./notes/Q42585641.md> Q42585641 </a></td>
    </tr>
    <tr>
      <th>59</th>
      <td>Will a biological database be different from a biological journal?</td>
      <td> <a href=./notes/Q21092584.md> Q21092584 </a></td>
    </tr>
    <tr>
      <th>60</th>
      <td>The future of annotation/biocuration</td>
      <td> <a href=./notes/Q56880438.md> Q56880438 </a></td>
    </tr>
    <tr>
      <th>61</th>
      <td>Swiss-Prot: juggling between evolution and stability.</td>
      <td> <a href=./notes/Q47579749.md> Q47579749 </a></td>
    </tr>
    <tr>
      <th>62</th>
      <td>Retinal ganglion cells: properties, types, genera, pathways and trans-species comparisons.</td>
      <td> <a href=./notes/Q48839876.md> Q48839876 </a></td>
    </tr>
    <tr>
      <th>63</th>
      <td>WikiPathways: connecting communities</td>
      <td> <a href=./notes/Q102205677.md> Q102205677 </a></td>
    </tr>
    <tr>
      <th>64</th>
      <td>The neuron classification problem</td>
      <td> <a href=./notes/Q36854753.md> Q36854753 </a></td>
    </tr>
    <tr>
      <th>65</th>
      <td>Petilla terminology: nomenclature of features of GABAergic interneurons of the cerebral cortex</td>
      <td> <a href=./notes/Q24620160.md> Q24620160 </a></td>
    </tr>
    <tr>
      <th>66</th>
      <td>Protein ontology on the semantic web for knowledge discovery</td>
      <td> <a href=./notes/Q100525924.md> Q100525924 </a></td>
    </tr>
    <tr>
      <th>67</th>
      <td>Towards a classification of stem cells</td>
      <td> <a href=./notes/Q64097544.md> Q64097544 </a></td>
    </tr>
    <tr>
      <th>68</th>
      <td>Reader engagement with medical content on Wikipedia</td>
      <td> <a href=./notes/Q90109096.md> Q90109096 </a></td>
    </tr>
    <tr>
      <th>69</th>
      <td>How Single-Cell Genomics Is Changing Evolutionary and Developmental Biology.</td>
      <td> <a href=./notes/Q38622289.md> Q38622289 </a></td>
    </tr>
    <tr>
      <th>70</th>
      <td>The evolution of cell types in animals: emerging principles from molecular studies.</td>
      <td> <a href=./notes/Q37301036.md> Q37301036 </a></td>
    </tr>
    <tr>
      <th>71</th>
      <td>Key transitions during the evolution of animal phototransduction: novelty, "tree-thinking," co-option, and co-duplication.</td>
      <td> <a href=./notes/Q46221687.md> Q46221687 </a></td>
    </tr>
    <tr>
      <th>72</th>
      <td>Neuronal diversity: too many cell types for comfort?</td>
      <td> <a href=./notes/Q48373178.md> Q48373178 </a></td>
    </tr>
    <tr>
      <th>73</th>
      <td>The BridgeDb framework: standardized access to gene, protein and metabolite identifier mapping services</td>
      <td> <a href=./notes/Q28842753.md> Q28842753 </a></td>
    </tr>
    <tr>
      <th>74</th>
      <td>Definitions</td>
      <td> <a href=./notes/Q102232524.md> Q102232524 </a></td>
    </tr>
    <tr>
      <th>75</th>
      <td>A recipe book for cell types in the human brain</td>
      <td> <a href=./notes/Q93087299.md> Q93087299 </a></td>
    </tr>
    <tr>
      <th>76</th>
      <td>Single-cell sequencing paints diverse pictures of the brain</td>
      <td> <a href=./notes/Q58084012.md> Q58084012 </a></td>
    </tr>
    <tr>
      <th>77</th>
      <td>Neuron Names: A Gene- and Property-Based Name Format, With Special Reference to Cortical Neurons</td>
      <td> <a href=./notes/Q64065346.md> Q64065346 </a></td>
    </tr>
    <tr>
      <th>78</th>
      <td>The BRAIN Initiative Cell Census Consortium: Lessons Learned toward Generating a Comprehensive Brain Cell Atlas.</td>
      <td> <a href=./notes/Q42778383.md> Q42778383 </a></td>
    </tr>
    <tr>
      <th>79</th>
      <td>Transcriptional landscape of the prenatal human brain</td>
      <td> <a href=./notes/Q33926910.md> Q33926910 </a></td>
    </tr>
    <tr>
      <th>80</th>
      <td>Nanopublications: A Growing Resource of Provenance-Centric Scientific Linked Data</td>
      <td> <a href=./notes/Q62924270.md> Q62924270 </a></td>
    </tr>
    <tr>
      <th>81</th>
      <td>Transcriptomic and morphophysiological evidence for a specialized human cortical GABAergic cell type</td>
      <td> <a href=./notes/Q60017992.md> Q60017992 </a></td>
    </tr>
    <tr>
      <th>82</th>
      <td>DisGeNET-RDF: harnessing the innovative power of the Semantic Web to explore the genetic basis of diseases</td>
      <td> <a href=./notes/Q28596617.md> Q28596617 </a></td>
    </tr>
    <tr>
      <th>83</th>
      <td>Cell-type-specific consequences of Reelin deficiency in the mouse neocortex, hippocampus, and amygdala</td>
      <td> <a href=./notes/Q28595082.md> Q28595082 </a></td>
    </tr>
    <tr>
      <th>84</th>
      <td>The Semanticscience Integrated Ontology (SIO) for biomedical research and knowledge discovery</td>
      <td> <a href=./notes/Q19671659.md> Q19671659 </a></td>
    </tr>
    <tr>
      <th>85</th>
      <td>Ontology Design Patterns for Semantic Web Content</td>
      <td> <a href=./notes/Q57774232.md> Q57774232 </a></td>
    </tr>
    <tr>
      <th>86</th>
      <td>Single-nucleus and single-cell transcriptomes compared in matched cortical cell types</td>
      <td> <a href=./notes/Q60933812.md> Q60933812 </a></td>
    </tr>
    <tr>
      <th>87</th>
      <td>How should access to a COVID-19 vaccine be regulated?</td>
      <td> <a href=./notes/Q102542608.md> Q102542608 </a></td>
    </tr>
    <tr>
      <th>88</th>
      <td>Shifting the paradigm: new approaches for characterizing and classifying neurons.</td>
      <td> <a href=./notes/Q37629171.md> Q37629171 </a></td>
    </tr>
    <tr>
      <th>89</th>
      <td>OWL 2 Web Ontology Language Document Overview (Second Edition)</td>
      <td> <a href=./notes/Q102926772.md> Q102926772 </a></td>
    </tr>
    <tr>
      <th>90</th>
      <td>OWL 2 Web Ontology Language Primer (Second Edition)</td>
      <td> <a href=./notes/Q102929379.md> Q102929379 </a></td>
    </tr>
    <tr>
      <th>91</th>
      <td>A genetic and computational approach to structurally classify neuronal types.</td>
      <td> <a href=./notes/Q34182035.md> Q34182035 </a></td>
    </tr>
    <tr>
      <th>92</th>
      <td>Adult mouse cortical cell taxonomy revealed by single cell transcriptomics</td>
      <td> <a href=./notes/Q27311570.md> Q27311570 </a></td>
    </tr>
    <tr>
      <th>93</th>
      <td>Identifiers.org - Compact Identifier Services in the Cloud</td>
      <td> <a href=./notes/Q100500142.md> Q100500142 </a></td>
    </tr>
    <tr>
      <th>94</th>
      <td>Immune-centric network of cytokines and cells in disease context identified by computational mining of PubMed.</td>
      <td> <a href=./notes/Q64962407.md> Q64962407 </a></td>
    </tr>
    <tr>
      <th>95</th>
      <td>Single-nucleus RNA-seq identifies Huntington disease astrocyte states</td>
      <td> <a href=./notes/Q89763874.md> Q89763874 </a></td>
    </tr>
    <tr>
      <th>96</th>
      <td>Converting neXtProt into Linked Data and nanopublications</td>
      <td> <a href=./notes/Q103834013.md> Q103834013 </a></td>
    </tr>
    <tr>
      <th>97</th>
      <td>Innovations present in the primate interneuron repertoire</td>
      <td> <a href=./notes/Q100395427.md> Q100395427 </a></td>
    </tr>
    <tr>
      <th>98</th>
      <td>A curated database reveals trends in single-cell transcriptomics</td>
      <td> <a href=./notes/Q103034964.md> Q103034964 </a></td>
    </tr>
    <tr>
      <th>99</th>
      <td>Species and cell-type properties of classically defined human and rodent neurons and glia</td>
      <td> <a href=./notes/Q57462271.md> Q57462271 </a></td>
    </tr>
    <tr>
      <th>100</th>
      <td>Single cell transcriptomics in neuroscience: cell classification and beyond</td>
      <td> <a href=./notes/Q88605805.md> Q88605805 </a></td>
    </tr>
    <tr>
      <th>101</th>
      <td>Towards the collaborative curation of the registry underlying Identifiers.org</td>
      <td> <a href=./notes/Q28709067.md> Q28709067 </a></td>
    </tr>
    <tr>
      <th>102</th>
      <td>The Covid-19 Conundrum</td>
      <td> <a href=./notes/Q103964322.md> Q103964322 </a></td>
    </tr>
    <tr>
      <th>103</th>
      <td>Towards the automatic classification of neurons</td>
      <td> <a href=./notes/Q27010194.md> Q27010194 </a></td>
    </tr>
    <tr>
      <th>104</th>
      <td>Brain structure. Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq.</td>
      <td> <a href=./notes/Q42776227.md> Q42776227 </a></td>
    </tr>
    <tr>
      <th>105</th>
      <td>Adding a little reality to building ontologies for biology</td>
      <td> <a href=./notes/Q21136172.md> Q21136172 </a></td>
    </tr>
    <tr>
      <th>106</th>
      <td>Data curation + process curation=data integration + science</td>
      <td> <a href=./notes/Q28303099.md> Q28303099 </a></td>
    </tr>
    <tr>
      <th>107</th>
      <td>Implementing a "publish, then review" model of publishing</td>
      <td> <a href=./notes/Q103811380.md> Q103811380 </a></td>
    </tr>
    <tr>
      <th>108</th>
      <td>Interneuron cell types are fit to function</td>
      <td> <a href=./notes/Q27021885.md> Q27021885 </a></td>
    </tr>
    <tr>
      <th>109</th>
      <td>Getting started in text mining</td>
      <td> <a href=./notes/Q21145382.md> Q21145382 </a></td>
    </tr>
    <tr>
      <th>110</th>
      <td>Abstract Meaning Representation for Sembanking</td>
      <td> <a href=./notes/Q104092178.md> Q104092178 </a></td>
    </tr>
    <tr>
      <th>111</th>
      <td>Reconstructing lineage hierarchies of the distal lung epithelium using single-cell RNA-seq</td>
      <td> <a href=./notes/Q34094294.md> Q34094294 </a></td>
    </tr>
    <tr>
      <th>112</th>
      <td>The genomic landscapes of individual melanocytes from human skin</td>
      <td> <a href=./notes/Q100491078.md> Q100491078 </a></td>
    </tr>
    <tr>
      <th>113</th>
      <td>Immune-evasive human islet-like organoids ameliorate diabetes</td>
      <td> <a href=./notes/Q98568692.md> Q98568692 </a></td>
    </tr>
    <tr>
      <th>114</th>
      <td>A single-cell transcriptome atlas of marsupial embryogenesis and X inactivation</td>
      <td> <a href=./notes/Q98568691.md> Q98568691 </a></td>
    </tr>
    <tr>
      <th>115</th>
      <td>Nanopublications for exposing experimental data in the life-sciences: a Huntington's Disease case study</td>
      <td> <a href=./notes/Q28606409.md> Q28606409 </a></td>
    </tr>
    <tr>
      <th>116</th>
      <td>Querying neXtProt nanopublications and their value for insights on sequence variants and tissue expression</td>
      <td> <a href=./notes/Q54670808.md> Q54670808 </a></td>
    </tr>
    <tr>
      <th>117</th>
      <td>Northstar enables automatic classification of known and novel cell types from tumor samples</td>
      <td> <a href=./notes/Q99564826.md> Q99564826 </a></td>
    </tr>
    <tr>
      <th>118</th>
      <td>Molecular biology gets wikified</td>
      <td> <a href=./notes/Q104161380.md> Q104161380 </a></td>
    </tr>
    <tr>
      <th>119</th>
      <td>Capillary cell-type specialization in the alveolus</td>
      <td> <a href=./notes/Q100560109.md> Q100560109 </a></td>
    </tr>
    <tr>
      <th>120</th>
      <td>Naming human genes</td>
      <td> <a href=./notes/Q98196903.md> Q98196903 </a></td>
    </tr>
    <tr>
      <th>121</th>
      <td>Guidelines for human gene nomenclature</td>
      <td> <a href=./notes/Q98148417.md> Q98148417 </a></td>
    </tr>
    <tr>
      <th>122</th>
      <td>Knowledge networks in the age of the Semantic Web</td>
      <td> <a href=./notes/Q28842577.md> Q28842577 </a></td>
    </tr>
    <tr>
      <th>123</th>
      <td>Internet encyclopaedias go head to head</td>
      <td> <a href=./notes/Q24239902.md> Q24239902 </a></td>
    </tr>
    <tr>
      <th>124</th>
      <td>snRNA-seq reveals a subpopulation of adipocytes that regulates thermogenesis</td>
      <td> <a href=./notes/Q101051363.md> Q101051363 </a></td>
    </tr>
    <tr>
      <th>125</th>
      <td>Beyond the frontiers of neuronal types.</td>
      <td> <a href=./notes/Q42156351.md> Q42156351 </a></td>
    </tr>
    <tr>
      <th>126</th>
      <td>Ontology Design Patterns for bio-ontologies: a case study on the Cell Cycle Ontology</td>
      <td> <a href=./notes/Q21093639.md> Q21093639 </a></td>
    </tr>
    <tr>
      <th>127</th>
      <td>Name-calling in the hippocampus (and beyond): coming to terms with neuron types and properties.</td>
      <td> <a href=./notes/Q37656899.md> Q37656899 </a></td>
    </tr>
    <tr>
      <th>128</th>
      <td>Biocuration - mapping resources and needs</td>
      <td> <a href=./notes/Q101217428.md> Q101217428 </a></td>
    </tr>
    <tr>
      <th>129</th>
      <td>Computing Machinery and Intelligence</td>
      <td> <a href=./notes/Q772056.md> Q772056 </a></td>
    </tr>
    <tr>
      <th>130</th>
      <td>ScispaCy: Fast and Robust Models for Biomedical Natural Language Processing</td>
      <td> <a href=./notes/Q101248419.md> Q101248419 </a></td>
    </tr>
    <tr>
      <th>131</th>
      <td>Hippocampome.org: a knowledge base of neuron types in the rodent hippocampus</td>
      <td> <a href=./notes/Q26253817.md> Q26253817 </a></td>
    </tr>
    <tr>
      <th>132</th>
      <td>An ontological approach to describing neurons and their relationships</td>
      <td> <a href=./notes/Q28730328.md> Q28730328 </a></td>
    </tr>
    <tr>
      <th>133</th>
      <td>Extracting a cellular hierarchy from high-dimensional cytometry data with SPADE</td>
      <td> <a href=./notes/Q34037159.md> Q34037159 </a></td>
    </tr>
    <tr>
      <th>134</th>
      <td>NeuroLex.org: an online framework for neuroscience knowledge</td>
      <td> <a href=./notes/Q28681367.md> Q28681367 </a></td>
    </tr>
    <tr>
      <th>135</th>
      <td>New insights into the classification and nomenclature of cortical GABAergic interneurons</td>
      <td> <a href=./notes/Q36747163.md> Q36747163 </a></td>
    </tr>
    <tr>
      <th>136</th>
      <td>Revisiting enigmatic cortical calretinin-expressing interneurons</td>
      <td> <a href=./notes/Q33796214.md> Q33796214 </a></td>
    </tr>
    <tr>
      <th>137</th>
      <td>Concept Wikiﬁcation for COVID-19</td>
      <td> <a href=./notes/Q104392113.md> Q104392113 </a></td>
    </tr>
    <tr>
      <th>138</th>
      <td>Local and Global Algorithms for Disambiguation to Wikipedia</td>
      <td> <a href=./notes/Q104415799.md> Q104415799 </a></td>
    </tr>
    <tr>
      <th>139</th>
      <td>Beyond the frontiers of neuronal types: fuzzy classification of interneurons.</td>
      <td> <a href=./notes/Q46137497.md> Q46137497 </a></td>
    </tr>
    <tr>
      <th>140</th>
      <td>Probabilistic cell typing enables fine mapping of closely related cell types in situ</td>
      <td> <a href=./notes/Q91330127.md> Q91330127 </a></td>
    </tr>
    <tr>
      <th>141</th>
      <td>mRNA-Seq whole-transcriptome analysis of a single cell</td>
      <td> <a href=./notes/Q28240611.md> Q28240611 </a></td>
    </tr>
    <tr>
      <th>142</th>
      <td>Entity linking for biomedical literature</td>
      <td> <a href=./notes/Q28642941.md> Q28642941 </a></td>
    </tr>
    <tr>
      <th>143</th>
      <td>SciBERT: A Pretrained Language Model for Scientific Text</td>
      <td> <a href=./notes/Q104433073.md> Q104433073 </a></td>
    </tr>
    <tr>
      <th>144</th>
      <td>Text mining approaches for dealing with the rapidly expanding literature on COVID-19</td>
      <td> <a href=./notes/Q104079663.md> Q104079663 </a></td>
    </tr>
    <tr>
      <th>145</th>
      <td>Structural Scaffolds for Citation Intent Classification in Scientific Publications</td>
      <td> <a href=./notes/Q104450390.md> Q104450390 </a></td>
    </tr>
    <tr>
      <th>146</th>
      <td>Integration of electrophysiological recordings with single-cell RNA-seq data identifies neuronal subtypes</td>
      <td> <a href=./notes/Q31033187.md> Q31033187 </a></td>
    </tr>
    <tr>
      <th>147</th>
      <td>Integrated Single-Cell Atlases Reveal an Oral SARS-CoV-2 Infection and Transmission Axis</td>
      <td> <a href=./notes/Q101210731.md> Q101210731 </a></td>
    </tr>
    <tr>
      <th>148</th>
      <td>Bridging the Gap Between Research and Practice: Predicting What Will Work Locally</td>
      <td> <a href=./notes/Q104475823.md> Q104475823 </a></td>
    </tr>
    <tr>
      <th>149</th>
      <td>Identification and characterization of distinct brown adipocyte subtypes in C57BL/6J mice</td>
      <td> <a href=./notes/Q103800764.md> Q103800764 </a></td>
    </tr>
    <tr>
      <th>150</th>
      <td>Dental cell type atlas reveals stem and differentiated cell types in mouse and human teeth</td>
      <td> <a href=./notes/Q99616616.md> Q99616616 </a></td>
    </tr>
    <tr>
      <th>151</th>
      <td>Construction of the Literature Graph in Semantic Scholar</td>
      <td> <a href=./notes/Q69587220.md> Q69587220 </a></td>
    </tr>
    <tr>
      <th>152</th>
      <td>CellMarker: a manually curated resource of cell markers in human and mouse</td>
      <td> <a href=./notes/Q56984510.md> Q56984510 </a></td>
    </tr>
    <tr>
      <th>153</th>
      <td>A standard nomenclature for referencing and authentication of pluripotent stem cells.</td>
      <td> <a href=./notes/Q48503347.md> Q48503347 </a></td>
    </tr>
    <tr>
      <th>154</th>
      <td>A resource for cell line authentication, annotation and quality control.</td>
      <td> <a href=./notes/Q38886793.md> Q38886793 </a></td>
    </tr>
    <tr>
      <th>155</th>
      <td>GeneDB and Wikidata</td>
      <td> <a href=./notes/Q91273803.md> Q91273803 </a></td>
    </tr>
    <tr>
      <th>156</th>
      <td>The Cellosaurus, a cell-line knowledge resource.</td>
      <td> <a href=./notes/Q54370168.md> Q54370168 </a></td>
    </tr>
    <tr>
      <th>157</th>
      <td>The bacterial species definition in the genomic era.</td>
      <td> <a href=./notes/Q33261434.md> Q33261434 </a></td>
    </tr>
    <tr>
      <th>158</th>
      <td>Theory of cell fate</td>
      <td> <a href=./notes/Q91908361.md> Q91908361 </a></td>
    </tr>
    <tr>
      <th>159</th>
      <td>Building a biomedical semantic network in Wikipedia with Semantic Wiki Links</td>
      <td> <a href=./notes/Q21169186.md> Q21169186 </a></td>
    </tr>
    <tr>
      <th>160</th>
      <td>Userscripts for the life sciences</td>
      <td> <a href=./notes/Q27062487.md> Q27062487 </a></td>
    </tr>
    <tr>
      <th>161</th>
      <td>biocuration</td>
      <td> <a href=./notes/biocuration.md> biocuration </a></td>
    </tr>
    <tr>
      <th>162</th>
      <td>SemMedDB: a PubMed-scale repository of biomedical semantic predications</td>
      <td> <a href=./notes/Q36432186.md> Q36432186 </a></td>
    </tr>
    <tr>
      <th>163</th>
      <td>Wikidata as a linked-data hub for Biodiversity data</td>
      <td> <a href=./notes/Q66058247.md> Q66058247 </a></td>
    </tr>
    <tr>
      <th>164</th>
      <td>Forcing cells to change lineages.</td>
      <td> <a href=./notes/Q34020397.md> Q34020397 </a></td>
    </tr>
    <tr>
      <th>165</th>
      <td>Stem cell states, fates, and the rules of attraction.</td>
      <td> <a href=./notes/Q37475461.md> Q37475461 </a></td>
    </tr>
    <tr>
      <th>166</th>
      <td>Integrated Morphoelectric and Transcriptomic Classification of Cortical GABAergic Cells</td>
      <td> <a href=./notes/Q102056745.md> Q102056745 </a></td>
    </tr>
    <tr>
      <th>167</th>
      <td>Common Cell type Nomenclature for the mammalian brain: A systematic, extensible convention</td>
      <td> <a href=./notes/Q104247451.md> Q104247451 </a></td>
    </tr>
    <tr>
      <th>168</th>
      <td>Neuro-symbolic representation learning on biological knowledge graphs.</td>
      <td> <a href=./notes/Q38377458.md> Q38377458 </a></td>
    </tr>
    <tr>
      <th>169</th>
      <td>Structured reviews for data and knowledge-driven research</td>
      <td> <a href=./notes/Q91866899.md> Q91866899 </a></td>
    </tr>
    <tr>
      <th>170</th>
      <td>Data management made simple</td>
      <td> <a href=./notes/Q56524364.md> Q56524364 </a></td>
    </tr>
    <tr>
      <th>171</th>
      <td>Everyone needs a data-management plan</td>
      <td> <a href=./notes/Q56524391.md> Q56524391 </a></td>
    </tr>
    <tr>
      <th>172</th>
      <td>Neuronal cell-type classification: challenges, opportunities and the path forward.</td>
      <td> <a href=./notes/Q46326031.md> Q46326031 </a></td>
    </tr>
    <tr>
      <th>173</th>
      <td>Naming of neurones. Classification and naming of cat retinal ganglion cells.</td>
      <td> <a href=./notes/Q41052480.md> Q41052480 </a></td>
    </tr>
    <tr>
      <th>174</th>
      <td>A rose by any other name... on 'naming of neurones' by Rowe and Stone.</td>
      <td> <a href=./notes/Q39790383.md> Q39790383 </a></td>
    </tr>
    <tr>
      <th>175</th>
      <td>Describing species: practical taxonomic procedure for biologists</td>
      <td> <a href=./notes/Q104043701.md> Q104043701 </a></td>
    </tr>
    <tr>
      <th>176</th>
      <td>Searching large-scale scRNA-seq databases via unbiased cell embedding with Cell BLAST</td>
      <td> <a href=./notes/Q97529981.md> Q97529981 </a></td>
    </tr>
    <tr>
      <th>177</th>
      <td>CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing</td>
      <td> <a href=./notes/Q64237155.md> Q64237155 </a></td>
    </tr>
    <tr>
      <th>178</th>
      <td>Probabilistic gene expression signatures identify cell-types from single cell RNA-seq data</td>
      <td> <a href=./notes/Q104371272.md> Q104371272 </a></td>
    </tr>
    <tr>
      <th>179</th>
      <td>CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing</td>
      <td> <a href=./notes/Q92937241.md> Q92937241 </a></td>
    </tr>
    <tr>
      <th>180</th>
      <td>Single-cell RNAseq cell cluster labelling by reference (celaref)</td>
      <td> <a href=./notes/Q104552099.md> Q104552099 </a></td>
    </tr>
    <tr>
      <th>181</th>
      <td>PanglaoDB: a web server for exploration of mouse and human single-cell RNA sequencing data</td>
      <td> <a href=./notes/Q63664483.md> Q63664483 </a></td>
    </tr>
    <tr>
      <th>182</th>
      <td>Unifying single-cell annotations based on the Cell Ontology</td>
      <td> <a href=./notes/Q104057222.md> Q104057222 </a></td>
    </tr>
    <tr>
      <th>183</th>
      <td>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</td>
      <td> <a href=./notes/Q83231039.md> Q83231039 </a></td>
    </tr>
    <tr>
      <th>184</th>
      <td>MARS: discovering novel cell types across heterogeneous single-cell experiments</td>
      <td> <a href=./notes/Q100737318.md> Q100737318 </a></td>
    </tr>
    <tr>
      <th>185</th>
      <td>Eleven grand challenges in single-cell data science</td>
      <td> <a href=./notes/Q89599719.md> Q89599719 </a></td>
    </tr>
    <tr>
      <th>186</th>
      <td>A comparison of automatic cell identification methods for single-cell RNA sequencing data</td>
      <td> <a href=./notes/Q89998341.md> Q89998341 </a></td>
    </tr>
    <tr>
      <th>187</th>
      <td>scMontage: Fast and Robust Gene Expression Similarity Search for Massive Single-cell Data</td>
      <td> <a href=./notes/Q104552207.md> Q104552207 </a></td>
    </tr>
    <tr>
      <th>188</th>
      <td>Current best practices in single-cell RNA-seq analysis: a tutorial.</td>
      <td> <a href=./notes/Q64974172.md> Q64974172 </a></td>
    </tr>
    <tr>
      <th>189</th>
      <td>Revealing the vectors of cellular identity with single-cell genomics.</td>
      <td> <a href=./notes/Q39002947.md> Q39002947 </a></td>
    </tr>
    <tr>
      <th>190</th>
      <td>CellO: comprehensive and hierarchical cell type classification of human cells with the Cell Ontology</td>
      <td> <a href=./notes/Q104552259.md> Q104552259 </a></td>
    </tr>
    <tr>
      <th>191</th>
      <td>Single-cell RNA-seq reveals new types of human blood dendritic cells, monocytes, and progenitors.</td>
      <td> <a href=./notes/Q34555562.md> Q34555562 </a></td>
    </tr>
    <tr>
      <th>192</th>
      <td>Single-cell sequencing of the human midbrain reveals glial activation and a neuronal state specific to Parkinson's disease</td>
      <td> <a href=./notes/Q104552303.md> Q104552303 </a></td>
    </tr>
    <tr>
      <th>193</th>
      <td>Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets</td>
      <td> <a href=./notes/Q34044099.md> Q34044099 </a></td>
    </tr>
    <tr>
      <th>194</th>
      <td>Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris</td>
      <td> <a href=./notes/Q59070958.md> Q59070958 </a></td>
    </tr>
    <tr>
      <th>195</th>
      <td>Comprehensive single-cell transcriptional profiling of a multicellular organism.</td>
      <td> <a href=./notes/Q38619221.md> Q38619221 </a></td>
    </tr>
    <tr>
      <th>196</th>
      <td>Broad transcriptional dysregulation of brain and choroid plexus cell types with COVID-19</td>
      <td> <a href=./notes/Q104552382.md> Q104552382 </a></td>
    </tr>
    <tr>
      <th>197</th>
      <td>A Single-Cell Atlas of In Vivo Mammalian Chromatin Accessibility</td>
      <td> <a href=./notes/Q59770872.md> Q59770872 </a></td>
    </tr>
    <tr>
      <th>198</th>
      <td>Cell type discovery using single-cell transcriptomics: implications for ontological representation.</td>
      <td> <a href=./notes/Q52625486.md> Q52625486 </a></td>
    </tr>
    <tr>
      <th>199</th>
      <td>ontoProc: processing of ontologies of anatomy, cell lines, and so on</td>
      <td> <a href=./notes/Q101074371.md> Q101074371 </a></td>
    </tr>
    <tr>
      <th>200</th>
      <td>Cells in experimental life sciences - challenges and solution to the rapid evolution of knowledge.</td>
      <td> <a href=./notes/Q48519854.md> Q48519854 </a></td>
    </tr>
    <tr>
      <th>201</th>
      <td>Cells in ExperimentaL Life Sciences (CELLS-2018): capturing the knowledge of normal and diseased cells with ontologies.</td>
      <td> <a href=./notes/Q64928913.md> Q64928913 </a></td>
    </tr>
    <tr>
      <th>202</th>
      <td>Reporting and connecting cell type names and gating definitions through ontologies</td>
      <td> <a href=./notes/Q91614095.md> Q91614095 </a></td>
    </tr>
    <tr>
      <th>203</th>
      <td>An ontology for cell types</td>
      <td> <a href=./notes/Q21184168.md> Q21184168 </a></td>
    </tr>
    <tr>
      <th>204</th>
      <td>Logical development of the cell ontology.</td>
      <td> <a href=./notes/Q33786317.md> Q33786317 </a></td>
    </tr>
    <tr>
      <th>205</th>
      <td>An improved ontological representation of dendritic cells as a paradigm for all cell types</td>
      <td> <a href=./notes/Q21284365.md> Q21284365 </a></td>
    </tr>
    <tr>
      <th>206</th>
      <td>Hematopoietic cell types: prototype for a revised cell ontology</td>
      <td> <a href=./notes/Q33940690.md> Q33940690 </a></td>
    </tr>
    <tr>
      <th>207</th>
      <td>Usage of cell nomenclature in biomedical literature.</td>
      <td> <a href=./notes/Q47565472.md> Q47565472 </a></td>
    </tr>
    <tr>
      <th>208</th>
      <td>Cell ontology in an age of data-driven cell classification.</td>
      <td> <a href=./notes/Q49192555.md> Q49192555 </a></td>
    </tr>
    <tr>
      <th>209</th>
      <td>flowCL: ontology-based cell population labelling in flow cytometry</td>
      <td> <a href=./notes/Q23809269.md> Q23809269 </a></td>
    </tr>
    <tr>
      <th>210</th>
      <td>CLO: The cell line ontology</td>
      <td> <a href=./notes/Q28650204.md> Q28650204 </a></td>
    </tr>
    <tr>
      <th>211</th>
      <td>Formalization of taxon-based constraints to detect inconsistencies in annotation and ontology development.</td>
      <td> <a href=./notes/Q33727235.md> Q33727235 </a></td>
    </tr>
    <tr>
      <th>212</th>
      <td>Toward principles for the design of ontologies used for knowledge sharing?</td>
      <td> <a href=./notes/Q47303277.md> Q47303277 </a></td>
    </tr>
    <tr>
      <th>213</th>
      <td>What Is Your Conceptual Definition of "Cell Type" in the Context of a Mature Organism?</td>
      <td> <a href=./notes/Q87649649.md> Q87649649 </a></td>
    </tr>
    <tr>
      <th>214</th>
      <td>Exciting times to study the identity and evolution of cell types</td>
      <td> <a href=./notes/Q90182085.md> Q90182085 </a></td>
    </tr>
    <tr>
      <th>215</th>
      <td>Function first: classifying cell types and circuits of the retina</td>
      <td> <a href=./notes/Q93189154.md> Q93189154 </a></td>
    </tr>
    <tr>
      <th>216</th>
      <td>Defining cell types and states with single-cell genomics</td>
      <td> <a href=./notes/Q26782737.md> Q26782737 </a></td>
    </tr>
    <tr>
      <th>217</th>
      <td>Cell type discovery and representation in the era of high-content single cell phenotyping.</td>
      <td> <a href=./notes/Q47870911.md> Q47870911 </a></td>
    </tr>
    <tr>
      <th>218</th>
      <td>Defining cellular identity through network biology.</td>
      <td> <a href=./notes/Q42084918.md> Q42084918 </a></td>
    </tr>
    <tr>
      <th>219</th>
      <td>Mapping the transcriptional diversity of genetically and anatomically defined cell populations in the mouse brain</td>
      <td> <a href=./notes/Q64118290.md> Q64118290 </a></td>
    </tr>
    <tr>
      <th>220</th>
      <td>Terminal Selectors of Neuronal Identity.</td>
      <td> <a href=./notes/Q38771724.md> Q38771724 </a></td>
    </tr>
    <tr>
      <th>221</th>
      <td>Construction of a human cell landscape at single-cell level</td>
      <td> <a href=./notes/Q90665256.md> Q90665256 </a></td>
    </tr>
    <tr>
      <th>222</th>
      <td>The definition of cell type</td>
      <td> <a href=./notes/Q77807915.md> Q77807915 </a></td>
    </tr>
    <tr>
      <th>223</th>
      <td>The Human Cell Atlas: from vision to reality.</td>
      <td> <a href=./notes/Q47565008.md> Q47565008 </a></td>
    </tr>
    <tr>
      <th>224</th>
      <td>CELLPEDIA: a repository for human cell information for cell studies and differentiation analyses.</td>
      <td> <a href=./notes/Q35482351.md> Q35482351 </a></td>
    </tr>
    <tr>
      <th>225</th>
      <td>Human Cell Atlas and cell-type authentication for regenerative medicine</td>
      <td> <a href=./notes/Q99418657.md> Q99418657 </a></td>
    </tr>
    <tr>
      <th>226</th>
      <td>Developing Global Cellular Information Retrieval System with Minimum Reporting Guidelines on Cellular Data for Regenerative Medicine</td>
      <td> <a href=./notes/Q104553080.md> Q104553080 </a></td>
    </tr>
    <tr>
      <th>227</th>
      <td>The human body at cellular resolution: the NIH Human Biomolecular Atlas Program</td>
      <td> <a href=./notes/Q90617765.md> Q90617765 </a></td>
    </tr>
    <tr>
      <th>228</th>
      <td>An estimation of the number of cells in the human body.</td>
      <td> <a href=./notes/Q34037445.md> Q34037445 </a></td>
    </tr>
    <tr>
      <th>229</th>
      <td>Opinion: Why science needs philosophy</td>
      <td> <a href=./notes/Q63479773.md> Q63479773 </a></td>
    </tr>
    <tr>
      <th>230</th>
      <td>The meanings of 'function' in biology and the problematic case of de novo gene emergence</td>
      <td> <a href=./notes/Q83227592.md> Q83227592 </a></td>
    </tr>
    <tr>
      <th>231</th>
      <td>Philosophical bias is the one bias that science cannot avoid</td>
      <td> <a href=./notes/Q64097527.md> Q64097527 </a></td>
    </tr>
    <tr>
      <th>232</th>
      <td>The challenges of big data biology</td>
      <td> <a href=./notes/Q63241652.md> Q63241652 </a></td>
    </tr>
    <tr>
      <th>233</th>
      <td>From primal scenes to synthetic cells</td>
      <td> <a href=./notes/Q64097538.md> Q64097538 </a></td>
    </tr>
    <tr>
      <th>234</th>
      <td>Ontologies in biology: design, applications and future challenges.</td>
      <td> <a href=./notes/Q35670188.md> Q35670188 </a></td>
    </tr>
    <tr>
      <th>235</th>
      <td>Ontologies: Formalising biological knowledge for bioinformatics.</td>
      <td> <a href=./notes/Q38522811.md> Q38522811 </a></td>
    </tr>
    <tr>
      <th>236</th>
      <td>FR-Match: robust matching of cell type clusters from single cell RNA sequencing data using the Friedman-Rafsky non-parametric test</td>
      <td> <a href=./notes/Q103045800.md> Q103045800 </a></td>
    </tr>
    <tr>
      <th>237</th>
      <td>Overview of the Alliance for Cellular Signaling.</td>
      <td> <a href=./notes/Q34164191.md> Q34164191 </a></td>
    </tr>
    <tr>
      <th>238</th>
      <td>CellFinder: a cell data repository</td>
      <td> <a href=./notes/Q28660708.md> Q28660708 </a></td>
    </tr>
    <tr>
      <th>239</th>
      <td>HeLa, a new microbial species</td>
      <td> <a href=./notes/Q104591553.md> Q104591553 </a></td>
    </tr>
    <tr>
      <th>240</th>
      <td>A survey on annotation tools for the biomedical literature.</td>
      <td> <a href=./notes/Q38068801.md> Q38068801 </a></td>
    </tr>
    <tr>
      <th>241</th>
      <td>Keeping raw data in context.</td>
      <td> <a href=./notes/Q43589284.md> Q43589284 </a></td>
    </tr>
    <tr>
      <th>242</th>
      <td>What makes a gene name? Named entity recognition in the biomedical literature.</td>
      <td> <a href=./notes/Q51955678.md> Q51955678 </a></td>
    </tr>
    <tr>
      <th>243</th>
      <td>Zika virus: designate standardized names</td>
      <td> <a href=./notes/Q24247056.md> Q24247056 </a></td>
    </tr>
    <tr>
      <th>244</th>
      <td>Representing Specified Values in OWL: "value partitions" and "value sets"</td>
      <td> <a href=./notes/Q104601506.md> Q104601506 </a></td>
    </tr>
    <tr>
      <th>245</th>
      <td>CELDA -- an ontology for the comprehensive representation of cells in complex systems</td>
      <td> <a href=./notes/Q21284308.md> Q21284308 </a></td>
    </tr>
    <tr>
      <th>246</th>
      <td>Using ontologies to study cell transitions</td>
      <td> <a href=./notes/Q34026802.md> Q34026802 </a></td>
    </tr>
    <tr>
      <th>247</th>
      <td>Genetic identification of brain cell types underlying schizophrenia</td>
      <td> <a href=./notes/Q58462175.md> Q58462175 </a></td>
    </tr>
    <tr>
      <th>248</th>
      <td>Q104626424</td>
      <td> <a href=./notes/Q104626424.md> Q104626424 </a></td>
    </tr>
    <tr>
      <th>249</th>
      <td>Uniform resolution of compact identifiers for biomedical data.</td>
      <td> <a href=./notes/Q55053685.md> Q55053685 </a></td>
    </tr>
    <tr>
      <th>250</th>
      <td>Semantically enabling a genome-wide association study database</td>
      <td> <a href=./notes/Q28707570.md> Q28707570 </a></td>
    </tr>
    <tr>
      <th>251</th>
      <td>The Monarch Initiative in 2019: an integrative data and analytic platform connecting phenotypes to genotypes across species</td>
      <td> <a href=./notes/Q91179320.md> Q91179320 </a></td>
    </tr>
    <tr>
      <th>252</th>
      <td>A Survey of Bioinformatics Database and Software Usage through Mining the Literature</td>
      <td> <a href=./notes/Q28600864.md> Q28600864 </a></td>
    </tr>
    <tr>
      <th>253</th>
      <td>LifeMap Discovery™: the embryonic development, stem cells, and regenerative medicine research portal</td>
      <td> <a href=./notes/Q34846475.md> Q34846475 </a></td>
    </tr>
    <tr>
      <th>254</th>
      <td>Distinction of Neurons, Glia and Endothelial Cells in the Cerebral Cortex: An Algorithm Based on Cytological Features</td>
      <td> <a href=./notes/Q36193595.md> Q36193595 </a></td>
    </tr>
    <tr>
      <th>255</th>
      <td>The Molecular Pages of the mesotelencephalic dopamine consortium (DopaNet)</td>
      <td> <a href=./notes/Q24802073.md> Q24802073 </a></td>
    </tr>
    <tr>
      <th>256</th>
      <td>SC2disease: a manually curated database of single-cell transcriptome for human diseases</td>
      <td> <a href=./notes/Q100429893.md> Q100429893 </a></td>
    </tr>
    <tr>
      <th>257</th>
      <td>AliBaba: PubMed as a graph.</td>
      <td> <a href=./notes/Q43433584.md> Q43433584 </a></td>
    </tr>
    <tr>
      <th>258</th>
      <td>The language of gene ontology: a Zipf's law analysis</td>
      <td> <a href=./notes/Q28268340.md> Q28268340 </a></td>
    </tr>
    <tr>
      <th>259</th>
      <td>Applying Ontology Design Patterns in Bio-ontologies</td>
      <td> <a href=./notes/Q57002925.md> Q57002925 </a></td>
    </tr>
    <tr>
      <th>260</th>
      <td>The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells.</td>
      <td> <a href=./notes/Q41861103.md> Q41861103 </a></td>
    </tr>
    <tr>
      <th>261</th>
      <td>A maximum-likelihood approach for building cell-type trees by lifting</td>
      <td> <a href=./notes/Q35906597.md> Q35906597 </a></td>
    </tr>
    <tr>
      <th>262</th>
      <td>Cell-type phylogenetics and the origin of endometrial stromal cells.</td>
      <td> <a href=./notes/Q38904507.md> Q38904507 </a></td>
    </tr>
    <tr>
      <th>263</th>
      <td>Implications of Epigenetic Variability within a Cell Population for "Cell Type" Classification</td>
      <td> <a href=./notes/Q26770736.md> Q26770736 </a></td>
    </tr>
    <tr>
      <th>264</th>
      <td>Decentralized provenance-aware publishing with nanopublications</td>
      <td> <a href=./notes/Q57339063.md> Q57339063 </a></td>
    </tr>
    <tr>
      <th>265</th>
      <td>Trusty URIs: Verifiable, Immutable, and Permanent Digital Artifacts for Linked Data</td>
      <td> <a href=./notes/Q56915510.md> Q56915510 </a></td>
    </tr>
    <tr>
      <th>266</th>
      <td>Using the brainbow to trace lineages in early mouse development</td>
      <td> <a href=./notes/Q58836377.md> Q58836377 </a></td>
    </tr>
    <tr>
      <th>267</th>
      <td>A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function</td>
      <td> <a href=./notes/Q29547320.md> Q29547320 </a></td>
    </tr>
    <tr>
      <th>268</th>
      <td>The value of data</td>
      <td> <a href=./notes/Q22676713.md> Q22676713 </a></td>
    </tr>
    <tr>
      <th>269</th>
      <td>Knowledge.Bio: A Web application for exploring, building and sharing webs of biomedical relationships mined from PubMed</td>
      <td> <a href=./notes/Q57831016.md> Q57831016 </a></td>
    </tr>
    <tr>
      <th>270</th>
      <td>histoCAT: analysis of cell phenotypes and interactions in multiplex image cytometry data.</td>
      <td> <a href=./notes/Q38637165.md> Q38637165 </a></td>
    </tr>
    <tr>
      <th>271</th>
      <td>What is a glial cell?</td>
      <td> <a href=./notes/Q46388366.md> Q46388366 </a></td>
    </tr>
    <tr>
      <th>272</th>
      <td>Div-Seq: Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons.</td>
      <td> <a href=./notes/Q41068482.md> Q41068482 </a></td>
    </tr>
    <tr>
      <th>273</th>
      <td>Geometry of the Gene Expression Space of Individual Cells</td>
      <td> <a href=./notes/Q35688096.md> Q35688096 </a></td>
    </tr>
    <tr>
      <th>274</th>
      <td>A Survey and Classification of Controlled Natural Languages</td>
      <td> <a href=./notes/Q57402167.md> Q57402167 </a></td>
    </tr>
    <tr>
      <th>275</th>
      <td>Fish oil, Raynaud's syndrome, and undiscovered public knowledge</td>
      <td> <a href=./notes/Q69703977.md> Q69703977 </a></td>
    </tr>
    <tr>
      <th>276</th>
      <td>Effective mapping of biomedical text to the UMLS Metathesaurus: the MetaMap program</td>
      <td> <a href=./notes/Q29614363.md> Q29614363 </a></td>
    </tr>
    <tr>
      <th>277</th>
      <td>BioGPS and MyGene.info: organizing online, gene-centric information</td>
      <td> <a href=./notes/Q27575818.md> Q27575818 </a></td>
    </tr>
    <tr>
      <th>278</th>
      <td>Accurate estimation of cell-type composition from gene expression data</td>
      <td> <a href=./notes/Q91665384.md> Q91665384 </a></td>
    </tr>
    <tr>
      <th>279</th>
      <td>Robust enumeration of cell subsets from tissue expression profiles</td>
      <td> <a href=./notes/Q36539395.md> Q36539395 </a></td>
    </tr>
    <tr>
      <th>280</th>
      <td>Determining cell type abundance and expression from bulk tissues with digital cytometry</td>
      <td> <a href=./notes/Q64109527.md> Q64109527 </a></td>
    </tr>
    <tr>
      <th>281</th>
      <td>Evolution of Cellular Differentiation: From Hypotheses to Models</td>
      <td> <a href=./notes/Q98633613.md> Q98633613 </a></td>
    </tr>
    <tr>
      <th>282</th>
      <td>Utilizing the Wikidata system to improve the quality of medical content in Wikipedia in diverse languages: a pilot study</td>
      <td> <a href=./notes/Q21503276.md> Q21503276 </a></td>
    </tr>
    <tr>
      <th>283</th>
      <td>Cell Identity Codes: Understanding Cell Identity from Gene Expression Profiles using Deep Neural Networks</td>
      <td> <a href=./notes/Q64263662.md> Q64263662 </a></td>
    </tr>
    <tr>
      <th>284</th>
      <td>Genome-wide atlas of gene expression in the adult mouse brain</td>
      <td> <a href=./notes/Q26253934.md> Q26253934 </a></td>
    </tr>
    <tr>
      <th>285</th>
      <td>Group formation, relatedness, and the evolution of multicellularity</td>
      <td> <a href=./notes/Q86916694.md> Q86916694 </a></td>
    </tr>
    <tr>
      <th>286</th>
      <td>GiniClust2: a cluster-aware, weighted ensemble clustering method for cell-type detection.</td>
      <td> <a href=./notes/Q53748361.md> Q53748361 </a></td>
    </tr>
    <tr>
      <th>287</th>
      <td>Situating Wikipedia as a health information resource in various contexts: A scoping review</td>
      <td> <a href=./notes/Q85632863.md> Q85632863 </a></td>
    </tr>
    <tr>
      <th>288</th>
      <td>Crowdsourced assessment of common genetic contribution to predicting anti-TNF treatment response in rheumatoid arthritis.</td>
      <td> <a href=./notes/Q37201814.md> Q37201814 </a></td>
    </tr>
    <tr>
      <th>289</th>
      <td>An RDF/OWL knowledge base for query answering and decision support in clinical pharmacogenetics.</td>
      <td> <a href=./notes/Q38420432.md> Q38420432 </a></td>
    </tr>
    <tr>
      <th>290</th>
      <td>Introduction to Random Boolean Networks</td>
      <td> <a href=./notes/Q104815188.md> Q104815188 </a></td>
    </tr>
    <tr>
      <th>291</th>
      <td>A global network of biomedical relationships derived from text.</td>
      <td> <a href=./notes/Q52681328.md> Q52681328 </a></td>
    </tr>
    <tr>
      <th>292</th>
      <td>Literature Based Discovery: models, methods, and trends.</td>
      <td> <a href=./notes/Q38371706.md> Q38371706 </a></td>
    </tr>
    <tr>
      <th>293</th>
      <td>Identification of region-specific astrocyte subtypes at single cell resolution</td>
      <td> <a href=./notes/Q90091126.md> Q90091126 </a></td>
    </tr>
    <tr>
      <th>294</th>
      <td>Single-cell profiling identifies myeloid cell subsets with distinct fates during neuroinflammation</td>
      <td> <a href=./notes/Q91210236.md> Q91210236 </a></td>
    </tr>
    <tr>
      <th>295</th>
      <td>BioGPS: an extensible and customizable portal for querying and organizing gene annotation resources.</td>
      <td> <a href=./notes/Q21183895.md> Q21183895 </a></td>
    </tr>
    <tr>
      <th>296</th>
      <td>Literature-based discovery of new candidates for drug repurposing.</td>
      <td> <a href=./notes/Q38817541.md> Q38817541 </a></td>
    </tr>
    <tr>
      <th>297</th>
      <td>GiniClust: detecting rare cell types from single-cell gene expression data with Gini index.</td>
      <td> <a href=./notes/Q31111856.md> Q31111856 </a></td>
    </tr>
    <tr>
      <th>298</th>
      <td>Evolutionary trade-offs, Pareto optimality, and the geometry of phenotype space.</td>
      <td> <a href=./notes/Q54337677.md> Q54337677 </a></td>
    </tr>
    <tr>
      <th>299</th>
      <td>CA1 pyramidal cells have diverse biophysical properties, affected by development, experience, and aging</td>
      <td> <a href=./notes/Q41694511.md> Q41694511 </a></td>
    </tr>
    <tr>
      <th>300</th>
      <td>Single-cell messenger RNA sequencing reveals rare intestinal cell types.</td>
      <td> <a href=./notes/Q34490100.md> Q34490100 </a></td>
    </tr>
    <tr>
      <th>301</th>
      <td>Toward an epistemology of Wikipedia</td>
      <td> <a href=./notes/Q101955295.md> Q101955295 </a></td>
    </tr>
    <tr>
      <th>302</th>
      <td>Online tools to support literature-based discovery in the life sciences.</td>
      <td> <a href=./notes/Q36280460.md> Q36280460 </a></td>
    </tr>
    <tr>
      <th>303</th>
      <td>The distribution of cellular turnover in the human body</td>
      <td> <a href=./notes/Q104873201.md> Q104873201 </a></td>
    </tr>
    <tr>
      <th>304</th>
      <td>Revised Estimates for the Number of Human and Bacteria Cells in the Body</td>
      <td> <a href=./notes/Q34537872.md> Q34537872 </a></td>
    </tr>
    <tr>
      <th>305</th>
      <td>Literature mining for the biologist: from information retrieval to biological discovery</td>
      <td> <a href=./notes/Q30002370.md> Q30002370 </a></td>
    </tr>
    <tr>
      <th>306</th>
      <td>Overview of BioCreAtIvE: critical assessment of information extraction for biology</td>
      <td> <a href=./notes/Q21093643.md> Q21093643 </a></td>
    </tr>
    <tr>
      <th>307</th>
      <td>What is the total number of protein molecules per cell volume? A call to rethink some published values</td>
      <td> <a href=./notes/Q26782007.md> Q26782007 </a></td>
    </tr>
    <tr>
      <th>308</th>
      <td>Are We Really Vastly Outnumbered? Revisiting the Ratio of Bacterial to Host Cells in Humans.</td>
      <td> <a href=./notes/Q34511374.md> Q34511374 </a></td>
    </tr>
    <tr>
      <th>309</th>
      <td>UniProt: the universal protein knowledgebase in 2021</td>
      <td> <a href=./notes/Q102383737.md> Q102383737 </a></td>
    </tr>
    <tr>
      <th>310</th>
      <td>Evaluation of text data mining for database curation: lessons learned from the KDD Challenge Cup.</td>
      <td> <a href=./notes/Q30811199.md> Q30811199 </a></td>
    </tr>
    <tr>
      <th>311</th>
      <td>Are Platelets Cells? And if Yes, are They Immune Cells?</td>
      <td> <a href=./notes/Q28083403.md> Q28083403 </a></td>
    </tr>
    <tr>
      <th>312</th>
      <td>Extracellular vesicles or exosomes? On primacy, precision, and popularity influencing a choice of nomenclature</td>
      <td> <a href=./notes/Q93135668.md> Q93135668 </a></td>
    </tr>
    <tr>
      <th>313</th>
      <td>Q104904652</td>
      <td> <a href=./notes/Q104904652.md> Q104904652 </a></td>
    </tr>
    <tr>
      <th>314</th>
      <td>Rutabaga by any other name: extracting biological names.</td>
      <td> <a href=./notes/Q42599537.md> Q42599537 </a></td>
    </tr>
    <tr>
      <th>315</th>
      <td>Body cell mass: model development and validation at the cellular level of body composition.</td>
      <td> <a href=./notes/Q47579411.md> Q47579411 </a></td>
    </tr>
    <tr>
      <th>316</th>
      <td>Mitochondrial DNA molecules and virtual number of mitochondria per cell in mammalian cells.</td>
      <td> <a href=./notes/Q34168557.md> Q34168557 </a></td>
    </tr>
    <tr>
      <th>317</th>
      <td>An interactive system for finding complementary literatures: a stimulus to scientific discovery</td>
      <td> <a href=./notes/Q57372877.md> Q57372877 </a></td>
    </tr>
    <tr>
      <th>318</th>
      <td>Exploiting semantic relations for literature-based discovery.</td>
      <td> <a href=./notes/Q35725188.md> Q35725188 </a></td>
    </tr>
    <tr>
      <th>319</th>
      <td>The quantified cell</td>
      <td> <a href=./notes/Q34504200.md> Q34504200 </a></td>
    </tr>
    <tr>
      <th>320</th>
      <td>SnapShot: key numbers in biology.</td>
      <td> <a href=./notes/Q34124340.md> Q34124340 </a></td>
    </tr>
    <tr>
      <th>321</th>
      <td>BioNumbers--the database of key numbers in molecular and cell biology</td>
      <td> <a href=./notes/Q24643881.md> Q24643881 </a></td>
    </tr>
    <tr>
      <th>322</th>
      <td>One Knowledge Graph to Rule Them All? Analyzing the Differences Between DBpedia, YAGO, Wikidata &amp; co</td>
      <td> <a href=./notes/Q41078402.md> Q41078402 </a></td>
    </tr>
    <tr>
      <th>323</th>
      <td>An era of single-cell genomics consortia</td>
      <td> <a href=./notes/Q99418649.md> Q99418649 </a></td>
    </tr>
    <tr>
      <th>324</th>
      <td>The Human Cell Atlas: Technical approaches and challenges.</td>
      <td> <a href=./notes/Q48563763.md> Q48563763 </a></td>
    </tr>
    <tr>
      <th>325</th>
      <td>The automation of science.</td>
      <td> <a href=./notes/Q27937312.md> Q27937312 </a></td>
    </tr>
    <tr>
      <th>326</th>
      <td>Representation of probabilistic scientific knowledge.</td>
      <td> <a href=./notes/Q36786073.md> Q36786073 </a></td>
    </tr>
    <tr>
      <th>327</th>
      <td>An atlas of active enhancers across human cell types and tissues.</td>
      <td> <a href=./notes/Q34660689.md> Q34660689 </a></td>
    </tr>
    <tr>
      <th>328</th>
      <td>A promoter-level mammalian expression atlas</td>
      <td> <a href=./notes/Q34412203.md> Q34412203 </a></td>
    </tr>
    <tr>
      <th>329</th>
      <td>Cell type-specific genetic regulation of gene expression across human tissues</td>
      <td> <a href=./notes/Q99349840.md> Q99349840 </a></td>
    </tr>
    <tr>
      <th>330</th>
      <td>xCell: digitally portraying the tissue cellular heterogeneity landscape.</td>
      <td> <a href=./notes/Q46377913.md> Q46377913 </a></td>
    </tr>
    <tr>
      <th>331</th>
      <td>Identifying cell types to interpret scRNA-seq data: how, why and more possibilities</td>
      <td> <a href=./notes/Q90784336.md> Q90784336 </a></td>
    </tr>
    <tr>
      <th>332</th>
      <td>Models of global gene expression define major domains of cell type and tissue identity</td>
      <td> <a href=./notes/Q33558314.md> Q33558314 </a></td>
    </tr>
    <tr>
      <th>333</th>
      <td>Building a high-quality Human Cell Atlas</td>
      <td> <a href=./notes/Q105080655.md> Q105080655 </a></td>
    </tr>
    <tr>
      <th>334</th>
      <td>scMatch: a single-cell gene expression profile annotation tool using reference datasets</td>
      <td> <a href=./notes/Q91557912.md> Q91557912 </a></td>
    </tr>
    <tr>
      <th>335</th>
      <td>A protocol for adding knowledge to Wikidata: aligning resources on human coronaviruses</td>
      <td> <a href=./notes/Q105037759.md> Q105037759 </a></td>
    </tr>
    <tr>
      <th>336</th>
      <td>Automating drug discovery.</td>
      <td> <a href=./notes/Q47780833.md> Q47780833 </a></td>
    </tr>
    <tr>
      <th>337</th>
      <td>Conceptions of Good Science in Our Data-Rich World.</td>
      <td> <a href=./notes/Q55214531.md> Q55214531 </a></td>
    </tr>
    <tr>
      <th>338</th>
      <td>On the formalization and reuse of scientific research.</td>
      <td> <a href=./notes/Q34178056.md> Q34178056 </a></td>
    </tr>
    <tr>
      <th>339</th>
      <td>A subcellular map of the human proteome.</td>
      <td> <a href=./notes/Q48155823.md> Q48155823 </a></td>
    </tr>
    <tr>
      <th>340</th>
      <td>Representation, simulation, and hypothesis generation in graph and logical models of biological networks.</td>
      <td> <a href=./notes/Q37921686.md> Q37921686 </a></td>
    </tr>
    <tr>
      <th>341</th>
      <td>The EXACT description of biomedical protocols.</td>
      <td> <a href=./notes/Q38513299.md> Q38513299 </a></td>
    </tr>
    <tr>
      <th>342</th>
      <td>The Bari Manifesto: An interoperability framework for essential biodiversity variables</td>
      <td> <a href=./notes/Q60250756.md> Q60250756 </a></td>
    </tr>
    <tr>
      <th>343</th>
      <td>The Philosophy of Logical Atomism, Lecture 1: Facts and Propositions</td>
      <td> <a href=./notes/Q105105637.md> Q105105637 </a></td>
    </tr>
    <tr>
      <th>344</th>
      <td>Transcriptomes of the major human pancreatic cell types.</td>
      <td> <a href=./notes/Q37430849.md> Q37430849 </a></td>
    </tr>
    <tr>
      <th>345</th>
      <td>Variability and memory of protein levels in human cells.</td>
      <td> <a href=./notes/Q40205119.md> Q40205119 </a></td>
    </tr>
    <tr>
      <th>346</th>
      <td>scConsensus: combining supervised and unsupervised clustering for cell type identification in single-cell RNA sequencing data</td>
      <td> <a href=./notes/Q104450617.md> Q104450617 </a></td>
    </tr>
    <tr>
      <th>347</th>
      <td>Study of cell differentiation by phylogenetic analysis using histone modification data</td>
      <td> <a href=./notes/Q30841360.md> Q30841360 </a></td>
    </tr>
    <tr>
      <th>348</th>
      <td>Evaluation of text-mining systems for biology: overview of the Second BioCreative community challenge.</td>
      <td> <a href=./notes/Q34844839.md> Q34844839 </a></td>
    </tr>
    <tr>
      <th>349</th>
      <td>Q</td>
      <td> <a href=./notes/Q.md> Q </a></td>
    </tr>
    <tr>
      <th>350</th>
      <td>Topic pages: PLoS Computational Biology meets Wikipedia</td>
      <td> <a href=./notes/Q21092570.md> Q21092570 </a></td>
    </tr>
    <tr>
      <th>351</th>
      <td>Submit a Topic Page to PLOS Computational Biology and Wikipedia</td>
      <td> <a href=./notes/Q54655231.md> Q54655231 </a></td>
    </tr>
    <tr>
      <th>352</th>
      <td>Genomic and molecular control of cell type and cell type conversions.</td>
      <td> <a href=./notes/Q49436822.md> Q49436822 </a></td>
    </tr>
    <tr>
      <th>353</th>
      <td>Inferring cell type innovations by phylogenetic methods-concepts, methods, and limitations.</td>
      <td> <a href=./notes/Q40436539.md> Q40436539 </a></td>
    </tr>
    <tr>
      <th>354</th>
      <td>From Freebase to Wikidata: The Great Migration</td>
      <td> <a href=./notes/Q24074986.md> Q24074986 </a></td>
    </tr>
    <tr>
      <th>355</th>
      <td>What Makes a Good Collaborative Knowledge Graph: Group Composition and Quality in Wikidata</td>
      <td> <a href=./notes/Q38918394.md> Q38918394 </a></td>
    </tr>
    <tr>
      <th>356</th>
      <td>Single-cell sequencing-based technologies will revolutionize whole-organism science.</td>
      <td> <a href=./notes/Q34360540.md> Q34360540 </a></td>
    </tr>
    <tr>
      <th>357</th>
      <td>Model-based understanding of single-cell CRISPR screening</td>
      <td> <a href=./notes/Q64052031.md> Q64052031 </a></td>
    </tr>
    <tr>
      <th>358</th>
      <td>Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage</td>
      <td> <a href=./notes/Q91045816.md> Q91045816 </a></td>
    </tr>
    <tr>
      <th>359</th>
      <td>Strategic reading, ontologies, and the future of scientific publishing.</td>
      <td> <a href=./notes/Q33493284.md> Q33493284 </a></td>
    </tr>
    <tr>
      <th>360</th>
      <td>Geospatial data and Scholia</td>
      <td> <a href=./notes/Q50813856.md> Q50813856 </a></td>
    </tr>
  </tbody>
</table>