# Blogs to read
https://bioeticacomplexa.blogspot.com/2020/05/
https://bioeticacomplexa.blogspot.com/2020/07/
# Books to read
Huxley, J. S.: The new systematics (Oxford Univ. Press, Oxford 1940).

Popper, K. R.: The open society and its enemies; 4th ed., vol. 1 (Routledge & Kegan Paul, London 1962)
Antonio D. Descarte’s Error. Vintage Press, 2006; 104–105.

Lorhard, J. (1606). Diagraph of Metaphysic or Ontology.St. Gallen: Ogdoas scholastica Book 8. PublishedinSangalli.Trans.Sara L. Uckelman.

Popper, K. R. Objective Knowledge: An Evolutionary Approach. Oxford: Clarendon, 1975.


 Elton CS. 1927. Animal Ecology. https://doi.org/10.5962/bhl.title.7435 

Wagner GP. 2014. Homology, Genes, and Evolutionary Innovatoin.
Princeton, NJ: Princeton University Press
# Papers to read
https://plato.stanford.edu/archives/win2019/entries/truth-revision/
https://seop.illc.uva.nl/entries/models-science/
# Cell types




Q64106229
Q89717434
Q49436822
Q96576585
Q35688096
Q47661868
Q90360171
## Other
Learning for single-cell assignment

Moana: a robust and scalable cell
type classification framework for single-cell RNA-Seq data.

Circular RNA expression in human hematopoietic cells is widespread and cell-type specific

scReClassify: post hoc cell type classification of single-cell rNA-seq data

TOAST: improving reference-free cell composition estimation by cross-cell type differential analysis

Cell-type deconvolution in epigenome-wide association studies: a review and recommendations

The selection and function of cell type-specific enhancers.

Cell-type-resolved alternative splicing patterns in mouse liver

https://pubmed.ncbi.nlm.nih.gov/6574104/

J Cell Biol. 1969 Oct 1; 43(1): 32–39.
doi: 10.1083/jcb.43.1.32
PMCID: PMC2107835
PMID: 5824067
A SIMPLE MODEL OF A STEADY STATE DIFFERENTIATING CELL SYSTEM
S. I. Rubinow

Discovering Pathway and Cell Type Signatures in Transcriptomic Compendia with Machine Learning

Computational methods for the integrative analysis of single-cell data 

Evolution of cellular diversity in primary motor cortex of human, marmoset monkey, and mouse. bioRxiv. doi:10.1101/2020.03.31.016972


Fuzik J, Z. 2016. Integration of electrophysiological recordings with single-cell RNA-seq data identifies neuronal subtypes. Nat Biotechnol 34:175–183. doi:10.1038/nbt.3443

Gala R, 2019. A coupled autoencoder approach for multi-modal analysis of cell types In: Wallach H, Larochelle H, Beygelzimer A, d\textquotesingle Alché-Buc F, Fox E, Garnett R, editors. Advances in Neural Information Processing Systems 32. Curran Associates, Inc. pp. 9267–9276.

Geirsdottir L, Cross-Species Single-Cell Analysis Reveals Divergence of the Primate Microglia Program. Cell 179:1609–1622.e16. doi:10.1016/j.cell.2019.11.010

Haendel MA,  2014. Unification of multi-species vertebrate anatomy ontologies for comparative biology in Uberon. J Biomed Semantics 5:21. doi:10.1186/2041-1480-5-21

Yao Z,  2020b. A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation. bioRxiv. doi:10.1101/2020.03.30.015214
A community-based transcriptomics classification and nomenclature of neocortical cell types. arXiv [q-bioGN].

Zeng H, Sanes JR. 2017. Neuronal cell-type classification: challenges, opportunities and the path forward. Nat Rev Neurosci 18:530–546. doi:10.1038/nrn.2017.85



### Describing Species - Judith Winston
[@doi:10.1017/CBO9781107415324.004]

https://portal.brain-map.org/explore/classes

https://github.com/AllenInstitute/nomenclature

This is a collection of resources to be read and parsed.


Neves M, Damaschun A, Mah N, Lekschas F, Seltmann S, Stachelscheid H, Fontaine JF, Kurtz A, Leser U. Preliminary evaluation of the CellFinder literature curation pipeline for gene expression in kidney cells and anatomical parts. Database 2013, 2013:bat020. doi:10.1093/database/bat020

https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-18-supplement-17

Mapping the transcriptional diversity of genetically and anatomically defined cell populations in the mouse brain

In 1957, C.H. Waddington proposed a powerful metaphor for how differentiated cell types emerge from a single, totipotent cell (Waddington 1957). T

https://www.nature.com/scitable/blog/bio2.0/discovering_new_cell_types_one/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3844401/

https://github.com/kevinrue/Hancock

DE QUEIROZ, K.  1994.  Replacement of an essentialistic perspective on taxonomicdefinitions as exemplified by the definition of "Mammalia."  Syst. Biol. 43: 497-510
 
  Replacement of an Essentialistic Perspective on Taxonomic Definitions as Exemplified by the Definition of "Mammalia"
https://www.sciencedirect.com/science/article/abs/pii/S1369848601000036?via%3Dihub

https://www.pnas.org/content/105/51/20067

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2662812/

 Molecular taxonomy of major neuronal classes in the adult mouse forebrain 
\cite{sugino2006molecular

/Genes and homology in nervous system evolution: Comparing gene functions, expression patterns, and cell type molecular fingerprints   \cite{arendt2005genes

Musser, J. M. & Wagner, G. P. Character trees from transcriptome data: origin and individuation of morphological characters and the so-called ‘species signal’. J. Exp. Zool. 324, 588–604 (2015).
Introduces the phenomenon of concerted transcriptome evolution and shows how it influences the comparison of cell type transcriptomes

Mullen, A. C. et al. Master transcription factors determine cell-type-specific responses to TGF-ß signaling. Cell 147, 565–576 (2011). Demonstrates empirically that cell type terminal selectors are responsible for mediating the cell type-specific response to a common cellular signal.

Hobert, O. Terminal selectors of neuronal identity. Curr. Top. Dev. Biol. 116, 455–475 (2016). Outlines the important role of terminal selector genes in establishing and maintaining cell type identity.


Stefanakis, N., Carrera, I. & Hobert, O. Regulatory logic of pan-neuronal gene expression in C. elegans. Neuron 87, 733–750 (2015). A systematic investigation of how different neuronal cell types use both cell type-specific and more general regulatory information to express pan-neuronal genes.

Arendt, D. Genes and homology in nervous system evolution: comparing gene functions, expression patterns, and cell type molecular fingerprints. Theory Biosci. 124, 185-197 (2005). A conceptual review that discusses the different levels of homology in evolutionary developmental biology research; it introduces the concept of molecular fingerprints for cell type comparisons.
Up next: 

% https://www.the-scientist.com/cover-story/advancing-techniques-reveal-the-brains-impressive-diversity-30197
% https://www.sciencedirect.com/topics/medicine-and-dentistry/cell-type
% http://www.diva-portal.org/smash/get/diva2:942241/FULLTEXT01.pdf
% https://sciencing.com/specialized-cells-definition-types-examples-13718073.html
% https://www.nature.com/scitable/blog/bio2.0/discovering_new_cell_types_one/

https://senselab.med.yale.edu/NeuronDB

https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000691

### Discovery of new cell types:

JJordao MJC, Sankowski R, Brendecke SM, Sagar LG, Tai YH, Tay TL, Schramm E, Armbruster S, Hagemeyer N, et al. Science 2019;363(6425):eaat7554. https://doi.org/10.1126/science.aat7554.
23. Gan Y, Li N, Zou G, Xin Y, Guan J. Identification of cancer subtypes from single-cell RNA-seq data using a consensus clustering method. BMC Med Genet. 2018;11(Suppl 6):117.
24. Mrdjen D, Pavlovic A, Hartmann FJ, Schreiner B, Utz SG, Leung BP, Lelios I, Heppner FL, Kipnis J, Merkler D, et al. High-dimensional single-cell mapping of central nervous system immune cells reveals distinct myeloid subsets in health, aging, and disease. Immunity. 2018;48(2):380–95 e386.
25. Der E, Ranabothu S, Suryawanshi H, Akat KM, Clancy R, Morozov P, Kustagi M, Czuppa M, Izmirly P, Belmont HM, et al. Single cell RNA sequencing to dissect the molecular heterogeneity in lupus nephritis. JCI Insight. 2017;2(9).

26. Villani AC, Satija R, Reynolds G, Sarkizova S, Shekhar K, Fletcher J, Griesbeck M, Butler A, Zheng S, Lazo S, et al. Single-cell RNA-seq reveals new types of human blood dendritic cells, monocytes, and progenitors. Science. 2017; 356(6335

Stevens, C.F. (1998). Neuronal diversity: Too many cell types for comfort

Mapping the physical network of cellular interactions
# Biocuration


Q28529449
Q21198755
Q58336167
Q18507561

## No QID

Machine learning for rediscovering revolutionary ideas of the past
Scientific intuition inspired by machine learning generated hypotheses
Tacit Knowledge, Trust and the Q of Sapphire
Towards operational abduction from a
cognitive perspective

67.Swanson, D. R. Undiscovered public knowledge, ¿¿er. Q. 56:103-118, 1986.
https://www.journals.uchicago.edu/doi/abs/10.1086/601720

Undiscovered Public Knowledge: a Ten-Year Update 
https://www.aaai.org/Papers/KDD/1996/KDD96-051.pdf

WikiUMLS: Aligning UMLS to Wikipedia via Cross-lingual Neural Ranking

## Other



6. P. Bourne, PLoS Comput. Biol. 1, e34 (2005).
7. P. Murray-Rust, H. S. Rzepa, J. Digit. Inform. 5, issue 1 (2004)


Kuhn T, Royer L, Fuchs NE, Schroeder M. 2006. Improving text mining with controlled natural language: a case study for protein interations. In: Proceedings DILS’06. Berlin Heidelberg: Springer

Kuhn T, Barbano PE, Nagy ML, Krauthammer M. 2013. Broadening the scope of nanopublications. In: Proceedings ofESWC 2013. Berlin Heidelberg: Springer, 487–50

Biotea, semantics for Pubmed Central

Making Linked Data SPARQL with the InterMine Biological Data Warehouse

An RDF/OWL knowledge base for query answering and decision support in clinical pharmacogenetics.

À. Bravo, J. Piñero, N. Queralt-Rosinach, M. Rautschka, L.I. Furlong, Extraction of relations between genes and diseases from text and large-scale data analysis: implications for translational research, BMC Bioinformatics 16 (2015), 55
DisGeNET RDF: A Gene-Disease Association Linked Open Data Resource


https://sn-scigraph.figshare.com/
Querying biomedical Linked Data with natural language questions

A competency question-oriented approach for the transformation of semi-structured bioinformatics data into linked open da
Modularisation of Domain Ontologies Implemented in Description Logics and related formalisms including OWL 

Ontologies for the life sciences Steffen Schulze-Kremer and Barry Smith  http://ontology.buffalo.edu/smith/articles/Ontologies_life_sciencesEnc.pdf
Lmn-2 interacts with Elf-2. On the meaning of common statements in biomedical literature

http://bioportal.bioontology.org/annotatorplus

Kim, J. J. & Rebholz-Schuhmann, D. Categorization of services for seeking information in biomedical literature: a typology for improvement of practice. Brief. Bioinformat. 9, 452–465 (2008). This manuscript exploits assumptions and observations linked to search behaviour from users of Web pages to judge the information-seeking behaviour of scientists. It judges available text-mining tools according to these assumptions.
3.


Rzhetsky, A., Iossifov, I., Loh, J. M. & White, K. P. Microparadigms: chains of collective reasoning in publications about molecular interactions. Proc. Natl Acad. Sci. USA 103, 4940–4945 (2006). This article explores how authors report on their results and how the collection of reported facts can be traced, compared and evaluated against each other. It gives early indications of what results might be produced if we applied automatic reasoning to the information from scientific literature and other resources.

Rzhetsky, A., Seringhaus, M. & Gerstein, M. Seeking a new biology through text mining. Cell 134, 9–13 (2008). The authors argue that the exploitation of the scientific literature will serve as an additional resource for the generation of hypotheses and the validation of human- driven hypotheses

Shah, N. H. et al. Comparison of concept recognizers for building the open biomedical annotator. BMC Bioinformatics 10, S14 (2009).



Rebholz-Schuhmann, D., Arregui, M., Gaudan, S., Kirsch, H. & Jimeno, A. Text processing through web services: calling Whatizit. Bioinformatics 24, 296–298 (2008).
1.  Shah, N. H. et al. Comparison of concept recognizers for building the open biomedical annotator. BMC Bioinformatics 10, S14 (2009).

Jessop, D. M., Adams, S. E., Willighagen, E. L., Hawizy, L. & Murray-Rust, P. OSCAR4: a flexible architecture for chemical text-mining. J. Cheminform. 3, 41 (2011).

Extraction of relations between genes and diseases from text and large-scale data analysis: implications for translational research.

Noy, N. F. et al. Bioportal: ontologies and integrated data resources at the click of a mouse. Nucleic Acids Res. 37, W170–W173 (2009

Himmelstein,D. Integrating resources with disparate licensing into an open network. https://thinklab.com/discussion/integrat ing-resources-with-disparate-licensing-into-an-open-network/ 107#1 (accessed Sep 16, 2016).

--
Neumann EK, Quan D: Biodash: a semantic web dashboard for drug development. Pac Symp on Biocomput 2006, 11:176-187.
--

Dumontier,M., Callahan,A., Cruz-Toledo,J., Ansell,P., Emonet,V. and Belleau,F. (2014) Bio2RDF release 3: a larger connected

https://academic.oup.com/nar/article/48/D1/D704/5614574

https://www.nejm.org/doi/full/10.1056/NEJMra1615014

 Ratnam,J., Zdrazil,B., Digles,D. et al. (2014) The applica- tion of the open pharmacological concepts triple store (open PHACTS) to support drug discovery research. PLOS ONE, 9, e115460.
6.

Lysenko,A., Roznov˘a¸t,I.A., Saqi,M. et al. (2016) Representing and querying disease networks using graph databases. BioData Min., 9, 23

A behind‐the‐scenes tour of the IEDB curation process: an optimized process empirically integrating automation and human curation efforts

Funding knowledgebases: Towards a sustainable funding model for the UniProt use case

https://academic.oup.com/bioinformatics/article/23/13/i41/238103
https://academic.oup.com/database/article/doi/10.1093/database/baaa006/5809234



https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1626147/
https://academic.oup.com/database/article/doi/10.1093/database/bar064/430006?searchresult=1

https://academic.oup.com/database/article/doi/10.1093/database/bau039/2634247?searchresult=1

https://academic.oup.com/database/article/doi/10.1093/database/baz085/5608006?searchresult=1
https://academic.oup.com/database/article/doi/10.1093/databa/baaa022/5828001

He Y, Yu H, Yang X, Shao C, Zhou W, Zhu Y, Wang H, Liu Q, Xie J, Zheng J, et al. Ontology: foundation of biomedical big data and precision medicine research. Chin J Bioinformatics. 2018;16(1):7–

On expert curation and scalability: UniProtKB/Swiss-Prot as a case study.

Guralnick RP, Cellinese N, Deck J, Pyle RL, Kunze J, Penev L, et al. Community next steps for making globally unique identifiers work for biocollections data. Zookeys. 2015; 133–154. https://doi.org/10. 3897/zookeys.494.9352 PMID: 25901117


Leitner F, Krallinger M, Rodriguez-Penagos C. Introducing meta-services for biomedical information extraction. Genome Biol 2008;9:S6


Crowdsourcing biomedical research: leveraging communities as innovation engines

Nonetheless, some examples of systems that support the integration of text mining with manual annotation are Textpresso [65], PaperBrowser [11], GOAnnotator [70] and PreBIND [74]


Mu¨ller H-M, Kenny EE, Sternberg PW. Textpresso: an ontology-based information retrieval and extraction system for biological literature. PLoS Biol 2004;2:e309.


Tá aqui os papers dos dois:
https://arxiv.org/abs/1902.07669
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3960092/

Tem esse serviço que tenta automatizar o processo com machine learning, mas eu acho bem primitivo ainda https://scite.ai/ ... Uma ontologia que todo mundo usasse ia ser ótimo

Arman Cohan, Waleed Ammar, Madeleine van Zuylen, and Field Cady. 2019. Structural scaffolds for ci- tation intent classification in scientific publications. CoRR, abs/1904.01608.
Nigel

Grenon P, Smith B, Goldberg L: Biodynamic Ontology: Applying BFO in the Biomedical Domain.In Ontologies in Medicine. Edited by Pisanelli DM. Amsterdam: Ios Press; 2004:20–38.
75.

1. Hakala, K., Kaewphan, S., Salakoski, T. & Ginter, F. Syntactic analyses and named entity recognition for PubMed and PubMed Central—up-to-the-minute. In Proceedings of the 15th Workshop on Biomedical Natural Language Processing 102–107, https://doi. org/10.18653/v1/W16-2913 (2016).
2. Bell, L., Chowdhary, R., Liu, J. S., Niu, X. & Zhang, J. Integrated bio-entity network: a system for biological knowledge discovery. PLoS One 6, e21474 (2011).

Lee, J. et al. BioBERT: a pre-trained biomedical language representation model for biomedical text mining. Bioinformatics 36, 1234–1240, https://doi.org/10.1093/bioinformatics/btz682 (2019).



BioGPS: building your own mash-up of gene annotations and expression profiles 
BioGPS and MyGene.info: organizing online, gene-centric information
High-performance web services for querying gene and variant annotation


## Reviews

Text mining resources for the life sciences

Pressing needs of biomedical text mining in biocuration and beyond: opportunities and challenges

## Text mining

Integrating text mining into the MGI biocuration workflow
Accelerating literature curation with text-mining tools: a case study of using PubTator to curate genes in PubMed abstracts
Collaborative biocuration--text-mining development task for document prioritization for curation
Discovering biomedical semantic relations in PubMed queries for information retrieval and database curation
Automated assessment of biological database assertions using the scientific literature
LitSense: making sense of biomedical literature at sentence level


## Ontology

Semi-automated ontology generation within OBO-Edit

The Confidence Information Ontology: a step towards a standard for asserting confidence in annotations

## Other
Knowledge engineering tools for reasoning with scientific observations and interpretations: a neural connectivity use case
Making your database available through Wikipedia: the pros and cons
Building a biomedical semantic network in Wikipedia with Semantic Wiki Links
MyMiner: a web application for computer-assisted biocuration and text annotation
GEOMetaCuration: a web-based application for accurate manual curation of Gene Expression Omnibus metadata
Accelerating annotation of articles via automated approaches: evaluation of the neXtA5 curation-support tool by neXtProt

## Biocreative

BioCreative III interactive task: an overview
Biocuration workflows and text mining: overview of the BioCreative 2012 Workshop Track II
An overview of the BioCreative 2012 Workshop Track III: interactive text mining task
Overview of the interactive task in BioCreative V
BioCreative V BioC track overview: collaborative biocurator assistant task for BioGRID
The BEL information extraction workflow (BELIEF): evaluation in the BioCreative V BEL and IAT track

## Crowd
Crowdsourcing in biomedicine: challenges and opportunities
Crowdsourcing and curation: perspectives from biology and natural language processing
Involving community in genes and pathway curation


# Top 100 papers
DNA sequencing with chain-terminating inhibitors.

Clustal W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.

Nonparametric estimation from incomplete observations.

Basic local alignment search tool.

Fuzzy sets.Zadeh, L. A. Inform. Control 8, 338–353 (1965).

Improved patch-clamp techniques for high-resolution current recording from cells and cell-free membrane patches. Hamill, O. P., Marty, A., Neher, E., Sakmann, B. & Sigworth, F. J.  Pflug. Arch. Eur. J. Physiol. 391, 85–100 (1981).

A rating scale for depression. Hamilton, M. J. Neurol. Neurosurg. Psychiatry 23, 56–62 (1960).

Maximum likelihood from incomplete data via EM algorithm. Dempster, A. P., Laird, N. M. & Rubin, D. B. J. R. Stat. Soc., B 39, 1–38 (1977).

Controlling the false discovery rate: a practical and powerful approach to multiple testing. Benjamini, Y. & Hochberg, Y.J. R. Stat. Soc. B 57, 289–300 (1995).

Isolation of mononuclear cells and granulocytes from human blood. Böyum, A. Scand. J. Clin. Lab. Invest. 21, S77–S89 (1968).

A new and rapid colorimetric determination of acetylcholinesterase activity. Ellman, G. L., Courtney, K. D., Andres, V. Jr & Featherstone, R. M.Biochem. Pharmacol. 7, 88 (1961).

The Protein Data Bank. Berman, H. M. et al. Nucleic Acids Res. 28, 235–242 (2000).

# Random

POINTS OF VIEW Visualizing biological data NATURE METHODS | VOL.9 NO.12 | DECEMBER 2012 | 1131
